k-es-demo-app/src/main/resources/db/migration/V1__Initial.sql

```diff
@@ -7,14 +7,14 @@ CREATE TABLE cmd
     serializationId nvarchar(255)  NOT NULL,
     error           BIT            NOT NULL,
     errorId         nvarchar(255),
-    data            nvarchar(4000) NOT NULL
+    data            nvarchar(max) NOT NULL
 )
 
 CREATE TABLE saga
 (
     correlationId   nvarchar(255)  NOT NULL,
     serializationId nvarchar(255)  NOT NULL,
-    data            nvarchar(4000) NOT NULL,
+    data            nvarchar(max) NOT NULL,
```

Comment:
Vet vi hva max er eller spiller det kanskje ikke noen rolle for oss?

Reply:
Sqlserver dokumentasjonen sier at max er best når `Use nvarchar(max) when the sizes of the column data entries vary considerably, and the string length might exceed 4,000 byte-pairs.` Greit nok